updateTablesToNewUser.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable guard-for-in */
+/* eslint-disable no-restricted-syntax */
+const AWS = require('aws-sdk');
+
+// Configure the AWS SDK with your credentials
+AWS.config.update({
+  accessKeyId: 'your_access_key',
+  secretAccessKey: 'your_secret_access_key',
+  region: 'your_region',
+});
+
+// Create an instance of the AWS Cognito Identity provider client
+const cognitoIdentityServiceProvider = new AWS.CognitoIdentityServiceProvider();
+
+// The ID of the Cognito User Pool you want to retrieve users from
+const userPoolId = 'us-east-1_lfq00ab2u';
+
+// Create an instance of the AWS DynamoDB client
+const dynamoDB = new AWS.DynamoDB();
+
+// The name of the DynamoDB table you want to update
+const exampleTableName1 = 'Receipt-example-staging';
+const exampleTableName2 = 'Vehicle-example-staging';
+const exampleTableName3 = 'User-example-staging';
+
+// Define a function to update the cognitoUsername field for a single entry in the DynamoDB table
+const updateCognitoItem = async (item, newSubID, tableName, fieldName) => {
+  const idIndex = Object.keys(item).findIndex((it) => it === 'id');
+
+  const params = {
+    TableName: tableName,
+    Key: {
+      // The primary key of the entry you want to update
+      [Object.keys(item)[idIndex]]: {
+        S: Object.values(item)[idIndex].S,
+      },
+    },
+    UpdateExpression: `SET ${fieldName} = :userID`,
+    ExpressionAttributeValues: {
+      ':userID': {
+        S: newSubID,
+      },
+    },
+    ReturnValues: 'UPDATED_NEW',
+  };
+
+  try {
+    // Call the DynamoDB updateItem method to update the cognitoUsername field
+    await dynamoDB.updateItem(params).promise();
+  } catch (err) {
+    console.log('This is a unsuccessful upload', err);
```

Comment:
This error message is not clear to me. What was being uploaded? It seems to me more like
```
update to dynamoDB item ${param.something} failed: err
```

Reply:
Made a better error message